### PR TITLE
config: port Domovoi schedule

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Ref. https://github.com/neo-project/neo/pull/3330.

### CODE REVIEW

File: config/protocol.mainnet.neofs.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ ProtocolConfiguration:
     Aspidochelone: 3000000
     Basilisk: 4500000
     Cockatrice: 5800000
+    Domovoi: 5900000
```
Comment (AnnaShaleva) [inline config/protocol.mainnet.neofs.yml:30]: ~10 days from the current 5838334. Is it enough or we need a smaller value?

Comment (roman-khimov) [reply config/protocol.mainnet.neofs.yml:30]: Doesn't really matter, can be 5800000 as well.

**Review by roman-khimov — APPROVED** [review]
OK, but waiting for C# to settle.

File: config/protocol.mainnet.neofs.yml (hunk 1 above)
Comment (AnnaShaleva) [reply config/protocol.mainnet.neofs.yml]: Fixed.

**Pull request merged**